openmm_md/plain_md.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<p>Optional: can we add a simple complex picture here?</p><ul><li>I could make one for you :)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Oh yes, that would be great if you can make a pretty picture for this! </p>

Reply:
added :)

Reply:
<p>Thank you, this looks great!</p>